docs/learn/learn-auction.md

```diff
@@ -34,11 +34,21 @@ When candle auctions are used online, they require a random number to decide the
 termination.
 
 Parachain slot auctions differ slightly from a normal candle auction in that it does not use
-the random number to decide the duration of its opening phase. Instead, it has a *known open phase*
-and will be retroactively determined (at the normal close) to have ended at some point in the past
-during the ending phase. So during the open phase, bids will continue to be accepted, but later bids
-have higher probability of losing since the retroactively determined close moment may be found to
-have preceded the time that a bid was submitted.
+the random number to decide the duration of its opening phase. 
+
+The candle auction on Polkadot is split into two main parts: the *opening period* which is in 
+effect immediately after the auction has started which is designed as a buffer-like time for parachain 
+candidates to setup their initial bids, and likely start executing their stragtey on how to win a slot 
+auction. So during the open phase, bids will continue to be accepted, but later bids have higher 
+probability of losing since the retroactively determined close moment may be found to have preceded the 
```

Comment:
Why wouldn't they?

Reply:
Like you already mentioned, it is a warm up phase where people come up with strategy to win the auction. The probability of winning kicks in once the ending period starts. Bids placed during opening period do not have effect the outcome. If you were referring to the ending period here, then it would have made sense. Ending period is mentioned in the next section of the article, as I see it.

Reply:
@DannyS03 Let me know if you are waiting for more input on this. I can request someone else to review this PR

Reply:
Gotcha, updated the content to clarify